tests/test_vllm_flash_attn.py

```diff
@@ -267,3 +267,63 @@ def test_varlen_with_paged_kv(
     )
     torch.testing.assert_close(output, ref_output, atol=2e-2, rtol=1e-2), \
         f"{torch.max(torch.abs(output - ref_output))}"
+
+@pytest.mark.parametrize("seq_lens", [(1023, 2049), (1023, 1023), (32, 32), (65, 65), (129, 129)])
```

Comment:
Thanks for adding the unit test! Can you please also include query_len=1 cases like (1, 1) and (1, 1024)? FA includes some special code paths for query_len=1, so it would be good to check here.

Reply:
Sure, added.